heft-plugins/heft-dev-cert-plugin/src/index.ts

```diff
@@ -12,6 +12,6 @@ import type { IHeftPlugin } from '@rushstack/heft';
 import { DevCertPlugin } from './DevCertPlugin';
 
 /**
- * @internal
+ * @public
```

Comment:
Why was this made public instead of `@internal`?

Reply:
Same with the other Heft plugins.

Reply:
@iclanton It used to be the case that it was marked as internal here and public in the api.md files, due to some strange undocumented behavior. Initially, this change fixed it so that the api.md files were listing these as internal and you said they should be public, so I made the change here to make them public. We have to decide whether we want to change it here to public or in the api.md files to internal